Code/GraphMol/Canon.cpp

```diff
@@ -1449,24 +1420,34 @@ class ChiralBondItem {
 
 class RankedValue {
  private:
-  OrderedSet<ChiralAtomItem> chiralAtomItems;
-  OrderedSet<ChiralBondItem> chiralBondItems;
+  std::vector<ChiralAtomItem> chiralAtomItems;
+  std::vector<ChiralBondItem> chiralBondItems;
 
  public:
   void AddAtom(Atom *atom, const std::vector<unsigned int> &atomsToInvert) {
-    chiralAtomItems.emplace(atom, atomsToInvert);
+    ChiralAtomItem chiralAtomItem(atom, atomsToInvert);
```

Comment:
as stated above, the check for pre-existence in the vector is unneccessary and this will be restored to a call to emplace_back.